# VDBFUSION
voxel_size: 0.05
sdf_trunc: 0.1
space_carving: False

# Reconstruction
fill_holes: True
min_weight: 5.0
